src/cli/commands/test/iac-local-execution/index.ts

```diff
@@ -0,0 +1,86 @@
+import * as fs from 'fs';
+import * as YAML from 'js-yaml';
+import { isLocalFolder } from '../../../../lib/detect';
+import { getFileType } from '../../../../lib/iac/iac-parser';
+import * as util from 'util';
+import { IacFileTypes } from '../../../../lib/iac/constants';
+import { IacFileScanResult, IacFileMetadata, IacFileData } from './types';
+import { buildPolicyEngine } from './policy-engine';
+import { formatResults } from './results-formatter';
+
+const readFileContentsAsync = util.promisify(fs.readFile);
+
+// this method executes the local processing engine and then formats the results to adapt with the CLI output.
+// the current version is dependant on files to be present locally which are not part of the source code.
+// without these files this method would fail.
+// if you're interested in trying out the experimnetal local execution model for IaC scanning, please reach-out.
```

Comment:
Nice. Reach out to who?

```suggestion
// this method executes the local processing engine and then formats the results to adapt with the CLI output.
// the current version is dependent on files to be present locally which are not part of the source code.
// without these files this method would fail.
// if you're interested in trying out the experimental local execution model for IaC scanning, please reach-out.
```